documentation/asciidoc/computers/camera/rpicam_options_still.adoc

```diff
@@ -1,148 +1,116 @@
-=== Still command line options
+=== Image options
 
-----
-	--quality,	-q		JPEG quality <number>
-----
+The command line options specified in this section apply only to still image output.
 
-Set the JPEG quality. Maximum quality is 100, and 93 is the default. Only applies when saving JPEG files.
+==== `quality`
 
-Example: `rpicam-jpeg -o test.jpg -q 80`
+Alias: `-q`
 
-----
-	--exif,		-x		Add extra EXIF tags <string>
-----
+Default value: `93`
 
-The extra EXIF tags are saved in the JPEG file. Only applies when saving JPEG files.
+Sets the JPEG quality. Accepts a value between `1` and `100`.
 
-EXIF is supported using the `libexif` library, so there are some associated limitations. In particular, `libexif` seems to recognise a number of tags, but without knowing the correct format for them. The software will currently treat these (incorrectly, in many cases) as ASCII, but will print a warning to the terminal. As we come across these they can be added to the table of known exceptions in the software.
+==== `exif`
 
-The application needs to supply EXIF tags that contain specific camera data (like the exposure time). But for other tags that have nothing to do with the camera, a reasonable workaround would simply be to add them _post facto_, using something like `exiftool`.
+Saves extra EXIF tags in the JPEG output file. Only applies to JPEG output. Because of limitations in the `libexif` library, many tags are currently (incorrectly) formatted as ASCII and print a warning in the terminal. Necessary for certain tags related to camera settings. You can instead add tags unrelated to camera settings to the output JPEG after recording with https://exiftool.org/[ExifTool].
```

Comment:
"Necessary for certain tags related to camera settings." -> "This option is necessary for adding certain EXIF tags related to camera settings." ? (and then perhaps remove the "instead" from the next sentence?)